tests/appveyor/mysql.appveyor.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+
+        <var name="db_type" value="pdo_mysql"/>
+        <var name="db_host" value="localhost" />
+        <var name="db_username" value="root" />
+        <var name="db_password" value="Password12!" />
+        <var name="db_name" value="doctrine_tests" />
+        <var name="db_port" value="3306"/>
+
+        <var name="tmpdb_type" value="pdo_mysql"/>
+        <var name="tmpdb_host" value="localhost" />
+        <var name="tmpdb_username" value="root" />
+        <var name="tmpdb_password" value="Password12!" />
+        <var name="tmpdb_port" value="3306"/>
+    </php>
+
+    <testsuites>
+        <testsuite name="Doctrine DBAL Test Suite">
+            <directory>C:\projects\dbal\tests\Doctrine\Tests\DBAL</directory>
```

Comment:
that was a change related to figuring out why the config files could not be read. I'll revert the related changes.